openjdk-8.yaml

```diff
@@ -0,0 +1,222 @@
+    with:
+      uri: https://icedtea.classpath.org/download/source/icedtea-3.27.0.tar.xz
+      expected-sha512: ee16aeb4f1564a1fb6132ed5165f925ca8edc8cef13b46291589ceaff42e8f9922f0834e4aba899da061e78ea48805e4ac5bba73a73d5389c05ba4efd1663812
+
+  - working-directory: /home/build/icedtea-drops
+    pipeline:
+      - uses: fetch
+        with:
+          uri: https://icedtea.classpath.org/download/drops/icedtea8/3.27.0/openjdk-git.tar.xz
+          expected-sha512: 227fa26be948f4405e3d8c363a7d196b66aa0c00139c2c4cbfda28ba8d554b6f983765eb0499a63bcae3c6166d5fa51499a3558fc5efee35d1d9b14bc1075aa1
+          extract: false
+      - uses: fetch
+        with:
+          uri: https://icedtea.classpath.org/download/drops/icedtea8/hotspot.tar.xz
+          expected-sha512: 5c8e839d2d74b0308f468d8763a69ab06920f43b92659bbac76f53b48d83ce8d7070a091ca36df99629364fc8d05b8272e7c12d04609b969d5e8db6ace6908df
+          extract: false
+
+  - runs: |
+      ./autogen.sh
+
+  - runs: |
+      export EXTRA_CFLAGS="$CFLAGS -std=gnu++98 -Wno-error -fno-delete-null-pointer-checks -fno-lifetime-dse -fno-strict-overflow"
+      export EXTRA_CPP_FLAGS="$CXXFLAGS -std=gnu++98 -fno-delete-null-pointer-checks -fno-lifetime-dse -fno-strict-overflow"
+
+      bash ./configure \
+        --build=${{host.triplet.gnu}} \
+        --host=${{host.triplet.gnu}} \
+        --prefix=/usr/lib/jvm/java-1.8-openjdk \
+        --sysconfdir=/etc \
+        --mandir=/usr/share/man \
+        --infodir=/usr/share/info \
+        --localstatedir=/var \
+        --with-parallel-jobs=$(nproc) \
+        --disable-dependency-tracking \
+        --disable-downloading \
+        --disable-precompiled-headers \
+        --disable-docs \
+        --disable-system-pcsc \
+        --disable-system-sctp \
+        --with-openjdk-src-zip=/home/build/icedtea-drops/openjdk-git.tar.xz \
+        --with-hotspot-src-zip=/home/build/icedtea-drops/hotspot.tar.xz \
+        --with-jdk-home=/usr/lib/jvm/java-1.7-openjdk \
+        --with-curves="nist+" \
+        --with-pkgversion="Wolfi ${{package.version}}-r${{package.epoch}}"
+
+      make icedtea-boot SHELL=/bin/bash
+      make
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/lib/jvm/java-1.8-openjdk
+      cp -a /home/build/openjdk.build/images/j2sdk-image/* "${{targets.destdir}}"/usr/lib/jvm/java-1.8-openjdk/
+      rm "${{targets.destdir}}"/usr/lib/jvm/java-1.8-openjdk/src.zip
+
+      # This archive contains absolute paths from the build environment,
+      # so it does not work on the target system. User can generate it
+      # running 'java -Xshare:dump'.
+      rm -f "${{targets.destdir}}"/usr/lib/jvm/java-1.8-openjdk/server/classes.jsa
+
+      # symlink to shared java cacerts store
+      rm -f "${{targets.destdir}}"/usr/lib/jvm/jvm-1.8-openjdk/jre/lib/security/cacerts
+      ln -sf /etc/ssl/certs/java/cacerts \
+        "${{targets.destdir}}"/usr/lib/jvm/java-1.8-openjdk/jre/lib/security/cacerts
+
+  - uses: strip
+
+subpackages:
```

Comment:
@kaniini I think will have suggestions related to -default-jvm

Reply:
the default-jvm for 7 was done as a subpackage, I can do the equivalent here if that's how we want to keep things!

https://github.com/wolfi-dev/os/blob/main/openjdk-7.yaml#L267-L277

Reply:
Yes, that's how all JVMs should be packaged in Wolfi moving forward.  We will do similar for dotnet when 6 lands.